app/views/content_items/specialist_document.html.erb

```diff
@@ -31,7 +31,9 @@
       <% end %>
     <% end %>
 
-    <%= render "govuk_publishing_components/components/contents_list_with_body", contents: @content_item.contents do %>
+    <% unless @content_item.hide_contents_list? %>
```

Comment:
This backwards logic is a bit painful. I think the method should be renamed to `show_contents_list?` so the check can be truthy.